app/views/layouts/application.html.erb

```diff
@@ -36,6 +36,7 @@
     </div>
 
     <div class="govuk-width-container">
+      <%= yield :before_content %>
```

Comment:
I wonder if we could call this content block `:back_link`. I suspect that'll be our only use for it.

What do you think?

Reply:
I've once again stolen this from Publish https://github.com/search?q=repo%3ADFE-Digital%2Fpublish-teacher-training%20before_content&type=code
As they use it for breadcrumbs (not sure yet if we are using breadcrumbs) I didn't think it was worth changing it.